invenio_db/cli.py

```diff
@@ -95,4 +95,8 @@ def destroy():
     """Drop database."""
     click.secho('Destroying database {0}'.format(_db.engine.url),
                 fg='red', bold=True)
-    drop_database(_db.engine.url)
+    try:
+        drop_database(_db.engine.url)
+    except FileNotFoundError as e:
+        click.secho('Sqlite database has not been initialised'
```

Comment:
You could pass `err`=True, it would print to standard error. What do you think?

Reply:
Since as we discussed it's more of a warning than an error I think it's fine to leave it like that for now.